src/tests/stdlib/strcat.c

```diff
@@ -0,0 +1,50 @@
+/**
+ * @file
+ *
+ * @date Mar 31, 2020
+ * @author: Prajawal Kumar Pandey
+ */
+
+#include <string.h>
+#include <stdlib.h>
+#include <embox/test.h>
+
+EMBOX_TEST_SUITE("test suite for strcat()");
+
+TEST_CASE("concatenate full string with strcat()") {
+    char src[] = "string";
+    char dst[] = "char ";
+
+    strcat(dst, src);
+    test_assert_not_null(dst);
```

Comment:
excess checking
dst cant become NULL anyway
Fix it in all places Please

Reply:
Updated Sir.

Reply:
Updated Sir.